release/src/main/groovy/TestScripts.groovy

```diff
@@ -95,16 +116,31 @@ class TestScripts {
      }
    }
 
-   // Check for expected results in stdout of the last command
+   // Check for expected results in stdout of the last command, if fails, log errors then exit.
    public void see(String expected) {
      if (!var.lastText.contains(expected)) {
        var.startDir.deleteDir()
        println "Cannot find ${expected} in ${var.lastText}"
-       _error("Cannot find expected text")
+       error("Cannot find expected text")
      }
      println "Verified $expected"
    }
 
+   public boolean seeOneOf(List<String> expected) {
```

Comment:
`seeAnyOf`? This succeeds on 1 or more matches

Reply:
done.